templates/apps/sumo-aws-apps/security-hub-org/security-hub-org-main.yaml

```diff
@@ -223,7 +255,10 @@ Parameters:
     
   LinkedRegions:
     Default: us-east-2, us-west-1
-    Description: Comma delimited list of regions. Findings from linked Regions can be viewed in the aggregation Region.
+    Description: 
+      "Comma delimited list of regions.
+      If RegionLinkingMode is ALL_REGIONS_EXCEPT_SPECIFIED, then this is a space-separated list of Regions that do not aggregate findings to the aggregation Region.
```

Comment:
If RegionLinkedMode is ALL_REGIONS, then Security Hub aggregates data from all Regions. Security Hub also aggregates data from new Regions as they are supported and you opt into them.

If RegionLinkingMode is SPECIFIED_REGIONS, then enter comma separated list of regions from where Security Hub aggregates findings to the aggregation Region. Security Hub does not aggregate data automatically from new Regions. 

Put the above description in Section4SecurityHubRegionLinkingMode and RegionLinkingMode parameters in both the templates.

For LinkedRegions and Section4SecurityHubLinkedRegions we can put below description

Comma delimited list of regions. Findings from linked Regions can be viewed in the aggregation Region. Required if the RegionLinkingMode is SPECIFIED_REGIONS.